### PR TITLE
Remove commentary on non-existent appx checks

### PR DESCRIPTION
**Pull request by riverar**
Discussion: #18630

### CODE REVIEW

File: src/vm/comcallablewrapper.cpp [hunk 1]
```diff
@@ -1924,16 +1924,8 @@ IUnknown* SimpleComCallWrapper::QIStandardInterface(REFIID riid)
...
-            // The AppX check was primarily done to ensure that we dont break VS in classic mode when it loads the desktop CLR since it needs
-            // objects to be non-Agile. In Apollo, we had objects agile in CoreCLR and even though we introduced AppX support in PhoneBlue,
-            // we should not constrain object agility using the desktop constraint, especially since VS does not rely on CoreCLR for its 
-            // desktop execution. 
-            //
-            // Keeping the Apollo behaviour also ensures that we allow SL 8.1 scenarios (which do not pass the AppX flag like the modern host) 
-            // to use CorDispatcher for async, in the expected manner, as the OS implementation for CoreDispatcher expects objects to be Agile.
+            // Don't implement IAgileObject if we are aggregated, if the object explicitly implements IMarshal, or if its ICustomQI returns
+            // Failed or Handled for IID_IMarshal (compat).
             if (!IsAggregated()
```
Comment (jkotas) [inline src/vm/comcallablewrapper.cpp]: Nit: Could you please change this to:

```
if (!IsAggregated())
{
```

while you are on it?

Comment (riverar) [reply src/vm/comcallablewrapper.cpp]: Done!

**Comment by jkotas** [comment]
@dotnet-bot test OSX10.12 x64 Checked Innerloop Build and Test please

**Comment by luqunl** [comment]
LGTM

**Pull request merged**